src/node_modules.cc

```diff
@@ -460,6 +465,28 @@ void BindingData::GetPackageScopeConfig(
     }
   }
 
+  return nullptr;
+}
+
+void BindingData::GetPackageScopeConfig(
+    const FunctionCallbackInfo<Value>& args) {
+  Realm* realm = Realm::GetCurrent(args);
+
+  bool invalid_url = false;
+  auto package_json = FindPackageScopeConfig(realm, args, &invalid_url);
```

Comment:
Really not a fan of this pattern. Would be nicer if `FindPackageScopeConfig` just returned either nullptr and we checked for that, or returned an `std::optional<T>` where "Invalid url" was signaled by the return value being `std::nullopt`.

Reply:
Updated 🙂